### PR TITLE
ENG-23448 - adding incident property to Incident type

### PR DESCRIPTION
**Pull request by juandagr**
US: [ENG-23448](https://alertlogic.atlassian.net/browse/ENG-23448?atlOrigin=eyJpIjoiNzAxZTM4ZTIxMGI0NGNmZTkyNDA1YzY3ZmVhOTg2NmMiLCJwIjoiaiJ9)

### CODE REVIEW

File: packages/iris/src/types/incident.class.ts [hunk 1]
```diff
@@ -82,6 +82,43 @@ export class VictimLset {
     }
 }
 
+interface IncidentPorperty {
```
Comment (fabiocsm) [inline packages/iris/src/types/incident.class.ts:85]: It should be IncidentProperties (plural and with no typos)

**Review by dadagama — APPROVED** [review]

**Comment by dadagama** [comment]
test it

**Review by fabiocsm — CHANGES_REQUESTED** [review]

**Review by fabiocsm — APPROVED** [review]
+1

**Comment by fabiocsm** [comment]
test it

**Pull request merged**